scipy/optimize/tests/test_cobyla.py

```diff
@@ -9,9 +9,18 @@
 from scipy.optimize import fmin_cobyla, minimize
 
 
+class Constraints(object):
+    def __init__(self, *args):
+        self.args = np.array(args[0])
+
+    def __call__(self, x):
+        return np.array([self.args[i, 0] * x[0] + self.args[i, 1] * x[1] +
+                        self.args[i, 2] for i in range(len(self.args))])
+
+
```

Comment:
Could you inline this as a function within test method as you did for other functions (`fun`, `cons2`)? It'd be more readable than having to jump outside to find out the definition.